components/admin_console/plugin_management/plugin_management.jsx

```diff
@@ -368,7 +368,10 @@ const PluginItem = ({
                 {')'}
             </div>
             {description}
-            <div className='padding-top'>
+            <div
+                className='padding-top'
+                id='pluginConfigSettings'
```

Comment:
This will cause Accessibility issue on "id attribute value must be unique" whenever there are 2 or more plugins.  My suggestion is to revert this and then add test ID on the parent [div](https://github.com/mattermost/mattermost-webapp/pull/4460/files#diff-5eac493766b80495371de8b80df7ada5R360) --> ``<div data-testid={pluginStatus.id}>``

Reply:
@saturninoabril , This is the to-do checklist while adding element. Thanks ,Will correct